configuration/sitemaps.md

```diff
@@ -537,16 +564,16 @@ Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
                                   >=25="orange", >=15="green", 0="white", <15="blue"]
 ```
 
-In the first example, the `Weather` Item is colored blue (label and value) if the `Temperature` is below or equal to 4°C.
+In the first example, the `Weather` Item is blue (label and value) if the `Temperature` is at or below 4°C.
 
-Expressions will be evaluated from left to right.
-The first condition returning `true` will decide on the color.
-Looking at the second example, you will see, that the given five expressions are ordered in the only meaningful combination.
+The order in which expressions are listed is important.
+Note that expressions are evaluated from left to right, and the first matching expression determines the color.
+Looking at the second example, you will see that the order of the expressions is set so that the first match, left-to-right gives the correct result.
+If the order was reversed, color assignment would not work properly.
```

Comment:
Not sure if this is clearer than the original.

Reply:
Hum - don't know what to say.  And now I do not have the original?

Reply:
You have, you can always check the git history of a file...

Reply:
For now, let's just go with this wording and move on.  I will figure out Git History at some other point.
This comment has been addressed.